### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by VrushtiJS**
Update the game over message so people know how to play again!

### CODE REVIEW

**Comment by jini55** [comment]
Yes, the changes are visible and ready to be merged.

File: README.md [hunk 1]
```diff
@@ -14,51 +14,45 @@ _Collaborate and work together on GitHub._
...
-  Include start button, a note about Actions minutes,
-  and tell the learner why they should take the course.
+  <<< Author notes: Step 3 >>>
+  Start this step by acknowledging the previous step.
+  Define terms and link to docs.github.com.
+  Historic note: this step combines the commend, approve, and needs changes steps from the previous version.
 -->
 
-## Welcome
+## Step 3: Leave a review
```
Comment (jini55) [inline README.md]: First review

**Review by jini55 — COMMENTED** [review]
Done, changes are visible.

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (VrushtiJS) [inline index.html:50]: added a suggestion...

File: README.md [hunk 3]
```diff
@@ -14,51 +14,23 @@ _Collaborate and work together on GitHub._
 </header>
 
 <!--
-  <<< Author notes: Course start >>>
-  Include start button, a note about Actions minutes,
-  and tell the learner why they should take the course.
+  <<< Author notes: Step 5 >>>
+  Start this step by acknowledging the previous step.
+  Define terms and link to docs.github.com.
 -->
```
Comment (VrushtiJS) [inline README.md:21]: heres another suggestion

File: README.md [hunk 4]
```diff
@@ -14,51 +14,23 @@ _Collaborate and work together on GitHub._
...
 
-## Welcome
```
Comment (VrushtiJS) [inline README.md:22]: suggestion

File: README.md [hunk 5]
```diff
@@ -14,51 +14,23 @@ _Collaborate and work together on GitHub._
 </header>
 
 <!--
-  <<< Author notes: Course start >>>
-  Include start button, a note about Actions minutes,
-  and tell the learner why they should take the course.
+  <<< Author notes: Step 5 >>>
+  Start this step by acknowledging the previous step.
```
Comment (VrushtiJS) [inline README.md]:
```suggestion
  Start this step by acknowledging the previous step. AND THIS IS A SUGGESTION
```

**Pull request merged**